Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,173 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <data>
-        <variable
-            name="email"
-            type="com.materialstudies.reply.data.Email" />
-    </data>
-
+  <data>
+    <variable
+        name="email"
+        type="com.materialstudies.reply.data.Email" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Can you change this to a CoordinatorLayout? A CoordinatorLayout will allow us to set a "behavior" on the NestedScrollView which automatically positions it below the AppBarLayout and will animate the app bar when scrolled.

I would also move the margin start and end to the NestedScrollView because with the margin as it is the AppBar's background gets inset and will look like this when scrolled:

<img width="494" alt="Screen Shot 2021-10-18 at 3 02 27 PM" src="https://user-images.githubusercontent.com/7459254/137791132-80f92b2f-8b3d-40b0-8173-efffc5f8b5a2.png">

The AppBarLayout "fills" it's background with color when something is scrolled underneath it.

Reply:
Also, you can just use `layout_marginHorizontal` if the start and end margins are the same